docs/ui-components/price-breakdown.md

```diff
@@ -0,0 +1,144 @@
+---
+layout: default
+title: Price Breakdown
+parent: UI Components
+nav_order: 4
+---
+
+# Price Breakdown
+{: .no_toc }
+
+<details open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+- TOC
+{:toc}
+</details>
+
+
+The price breakdown component displays information about Afterpay instalments and handles a number of common configurations.
+
+A total payment amount (represented as a `BigDecimal`) must be programatically set on the component to display Afterpay instalment information.
+
+```kotlin
+val totalAmount: BigDecimal = getTotalAmount()
+
+val paymentBreakdown = view.findViewById<AfterpayPriceBreakdown>(R.id.priceBreakdown)
+paymentBreakdown.totalAmount = totalAmount
+```
+
+When the breakdown component is assigned a total amount that is valid for the merchant account, the 4 instalment amounts will be displayed.
```

Comment:
```suggestion
When the breakdown component is assigned a total amount that is valid for the merchant account, the 4 installment amounts will be displayed.
```

Note that we can also use the term "payments" instead of "instalments/installments".